### DISCUSSION TITLE
Some functionality I found "missing"

### DISCUSSION THREAD

**Discussion by nagdav853** [uxbox/uxbox#593] [General]
As I already tooted about it on Mastodon, it's shaping up to be an amazing free and open source alternative to something like Figma, and I'm really impressed how feature-rich this alpha release is, compared to other FOSS design tools in the same state.

After using the app I have some observations, which some people before me might have voiced too - I'm sorry if I'm repeating what others already said. Some functions, which I use daily for my job are, and I found missing:
- Dragging guides from the rulers
- Intelligent duplication with Ctrl+D, where the direction and distance of the first duplication is added to the repetitions
- Export multiple screens or layers at once
- Distribute items, and change the padding between them from an input field in px
- Objects having multiple fills and/or strokes to achieve certain effects with blending modes
- Swapping the color of the fill and the stroke by pressing Shift+X
- Having images as fills, so I can add color overlays on top
- Boolean operations for vector shapes
- Using my local fonts
- Multiple polygon tools/options
- Ctrl+Shift+G ungrouping items
- Flipping items horizontally or vertically

I hope I'll get to see them in the future 😄 As I said, it's an amazing app, and I'm glad that I get to see a design software like this in the open source community! All the best to the team! 😄

**Comment by elhombretecla** [comment]
Thank you very much for your feedback!!

We're already planning our next sprints in terms of functionalities prioritization. We take all your comments into consideration :)
Thank you very much for being part of the Penpot community. You help is very welcome!
